docs/hw/hookup.md

```diff
@@ -3,6 +3,39 @@
 The iRobot® Create® 3 has a USB-C®[^1] connector implementing a USB 2.0 host capable of 5 V @ 3 A continuous, which can be used to power and communicate with various downstream devices.
 Here are some hookup examples.
 
+## NXP NavQ+
+Since NavQ+ has USB-C® ports capable of being configured as a USB Ethernet Gadget, a cable with a USB-C® connector on both ends is all that is required.
```

Comment:
Is NAVQ+ a trademark of some kind? Does it require attribution?

Reply:
It is not.

Reply:
In that case, maybe you don't need to rework this

Reply:
Ok, I'll leave it as is, but if you need it changed I can.